extern/tbb/CMakeLists.txt

```diff
@@ -1,11 +1,11 @@
 # TRACCC library, part of the ACTS project (R&D line)
 #
-# (c) 2022 CERN for the benefit of the ACTS project
+# (c) 2022-2024 CERN for the benefit of the ACTS project
 #
 # Mozilla Public License Version 2.0
 
 # CMake include(s).
-cmake_minimum_required( VERSION 3.14 )
+cmake_minimum_required( VERSION 3.24 )
```

Comment:
Isn't this quite recent? I'm a bit surprised all our CI machines have this!

Reply:
I was pleasantly surprised as well. This is what I need for `OVERRIDE_FIND_PACKAGE`, so was very happy that the CI accepted it without further effort. 😄